lua/wikis/chess/GetMatchGroupCopyPaste/wiki.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- wiki=chess
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---@class ChessMatch2CopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = WikiCopyPaste.Indent
+
+--returns the Code for a Match, depending on the input
+---@param bestof integer
+---@param mode string
+---@param index integer
+---@param opponents integer
+---@param args table
+---@return string
+function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
+	local showScore = bestof == 0
+	local opponent = WikiCopyPaste.getOpponent(mode, showScore)
+
+	local lines = Array.extend(
+		'{{Match',
+		showScore and (INDENT .. '|finished=') or nil,
+		INDENT .. '|date=',
+		Logic.readBool(args.streams) and (INDENT .. '|twitch=|youtube=|vod=') or nil,
+		Array.map(Array.range(1, opponents), function(opponentIndex)
+			return INDENT .. '|opponent' .. opponentIndex .. '=' .. opponent
+		end),
+		bestof ~= 0 and Array.map(Array.range(1, bestof), WikiCopyPaste._map) or nil,
```

Comment:
```suggestion
		Array.map(Array.range(1, bestof), WikiCopyPaste._map),
```
is enough (if bestof is 0 the Array.range returns empty table and hence Array.map returns an empty table too)